tutorials/multiple_robots.md

```diff
@@ -110,11 +131,11 @@ Once the Create® 3 application has restarted, try calling `ros2 topic list` on
 
 ## Namespacing
 
-Namespacing is a method of adding a prefix to topic names to group certain topics, or to make them unique. 
+Namespacing is a method of adding a prefix to topic names to group certain topics, or to make them unique.
```

Comment:
Superfluous new line or maybe this should be two paragraphs (flagged because the space was removed as part of that other discussion)